src/emonesp.h

```diff
@@ -5,15 +5,7 @@
 // General support code used by all modules
 // -------------------------------------------------------------------
 
-// Uncomment to use hardware UART 1 for debug else use UART 0
-// #define DEBUG_SERIAL1
```

Comment:
Yes, very sorry this was a mistake to commit this. Debug should not go to the main serial.